src/ert/_c_wrappers/enkf/_deprecation_migration_suggester.py

```diff
@@ -102,6 +105,15 @@ def add_suggestion(kw, suggestion):
                 "support for RHS queues. It no longer has any effect "
                 "and can safely be removed from the config file.",
             )
+        for kw in self.USE_QUEUE_OPTION:
+            add_suggestion(
+                kw,
+                f"The {kw} keyword have been removed. For most cases these options "
```

Comment:
Im not certain, but should 'have' been 'has' here?

Reply:
You have right.